netadapter/server/grpcserver/p2pserver.go

```diff
@@ -0,0 +1,44 @@
+package grpcserver
+
+import (
+	"github.com/kaspanet/kaspad/netadapter/server/grpcserver/protowire"
+	"github.com/kaspanet/kaspad/util/panics"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/status"
+)
+
+type p2pServer struct {
```

Comment:
Please rename p2pServer to something a bit more concrete. How about protocolServer?

Reply:
Talked about it in chat and decided to leave as is for now